apps/extension/server/src/utils.ts

```diff
@@ -11,6 +11,18 @@ export const isErrorWithMessage = (
   'message' in error &&
   typeof (error as Record<string, unknown>).message === 'string';
 
+export const getErrorMessage = (error: unknown): string => {
```

Comment:
If we want we can reuse the `isErrorWithMessage` helper here and do:
```ts
const getErrorMessage = (error: unknown): string => isErrorWithMessage(error) ? error.message : 'Unknown error';
```

If we don't want to do that we should just remove `isErrorWithMessage` entirely as it is not used anymore.

Reply:
Ah ye, my intention was to remove it - i forgot to do it. Thanks for noticing 👍